tasks.py.jinja

```diff
@@ -309,6 +354,115 @@ def start(c, detach=True, debugpy=False):
     _logger.info("Waiting for services to spin up...")
     time.sleep(SERVICES_WAIT_TIME)
 
+@task(
+    develop,
+    help={
+        "modules": "Comma-separated list of modules to test. Default: 'base'.",
+        "core": "Install all core addons. Default: False",
+        "extra": "Install all extra addons. Default: False",
+        "private": "Install all private addons. Default: False",
+    },
+)
+def install(c, modules=None, core=False, extra=False, private=False):
+    """Install Odoo addons"""
+    if not (modules or core or extra or private):
+        raise exceptions.ParseError(
+            message="You must provide at least one option. "
+            "See --help for details."
+        )
+    cmd = "docker-compose run --rm addons init"
+    if core:
+        cmd += " --core"
+    if extra:
+        cmd += " --extra"
+    if private:
+        cmd += " --private"
+    if modules:
+        cmd += f" -w {modules}"
+    with c.cd(str(PROJECT_ROOT)):
+        c.run(
+            cmd,
+            env=dict(
+                UID_ENV,
+            ),
+        )
+
+
+def _get_cwd_addon(file=None):
+    if not file:
+        cwd = Path.cwd()
+    else:
+        cwd = Path.parent(file)
+    manifest_file = False
+    while not manifest_file:
+        manifest_file = (
+            Path(cwd / "__manifest__.py").exists()
+            or Path(cwd / "__openerp__.py").exists()
+        )
+        if manifest_file:
+            return cwd.stem
+        cwd = cwd.parent
+        if cwd == PROJECT_ROOT:
+            return None
+
+
+@task(
+    develop,
+    help={
+        "debugpy": "Whether or not to run tests in a VSCode debugging session. "
+        "Default: False",
+        "modules": "Comma-separated list of modules to test. Default: 'base'.",
+        "logs": "Whether or not to follow Odoo logs during testing. " "Default: True",
+    },
+)
+def test(c, modules=None, debugpy=False, logs=True, cur_file=None):
+    """Run Odoo tests
+    NOTE: Odoo must be restarted manually after this to go back to normal mode
+    """
+    if not modules:
+        cur_module = _get_cwd_addon(file=cur_file)
+        modules = cur_module if cur_module is not None else "base"
```

Comment:
You don't need to be so specific in Python. Just:

```py
        modules = cur_module or "base"
```

However, as said above, I think "base" is not a good default.